bulkimport/src/main/java/com/microsoft/azure/documentdb/bulkimport/ExceptionUtils.java

```diff
@@ -25,7 +25,7 @@
 import com.microsoft.azure.documentdb.DocumentClientException;
 import com.microsoft.azure.documentdb.internal.HttpConstants;
 
-class ExceptionUtils {
+public class ExceptionUtils {
```

Comment:
why does this have to be public?

Reply:
Since it has to be accessible to the bulk update package. We're going to be adding delete functionality next too, so we need to separate the files into packages.